server/api/urls.py

```diff
@@ -8,9 +8,13 @@
 router.register(r'equipment-types', views.EquipmentTypeViewSet)
 router.register(r'equipment-items', views.EquipmentItemViewSet)
 router.register(r'equipment-categories', views.EquipmentCategoryViewSet)
+router.register(r'equipment-requests', views.EquipmentRequestViewSet)
 
 urlpatterns = [
     *router.urls,
-    re_path(r'equipment', views.list_equipment),
+    # re_path(r'equipment', views.list_equipment),
     path('snippets/', views.EquipmentTypeList.as_view()),
+    path('equipment-availability/<str:request_out>/<str:request_in>/<str:equipment_item_id>/', views.get_availability),
```

Comment:
To be consistent with [RESTful API endpoint naming conventions](https://restfulapi.net/resource-naming/), you would want to pass in these parameters as query strings. Something like`availability/?out=&in=&id=`, since the resource we are returning is an availability record.

Reply:
Agreed

Reply:
Fixed!